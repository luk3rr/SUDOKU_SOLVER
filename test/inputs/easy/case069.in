003000561 708060200 600000000 070020000 300680090 406073100 000000002 000508000 500400600 
