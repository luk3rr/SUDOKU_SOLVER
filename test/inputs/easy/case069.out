Solving the following grid:
0 0 3 | 0 0 0 | 5 6 1 
7 0 8 | 0 6 0 | 2 0 0 
6 0 0 | 0 0 0 | 0 0 0 
------+-------+------
0 7 0 | 0 2 0 | 0 0 0 
3 0 0 | 6 8 0 | 0 9 0 
4 0 6 | 0 7 3 | 1 0 0 
------+-------+------
0 0 0 | 0 0 0 | 0 0 2 
0 0 0 | 5 0 8 | 0 0 0 
5 0 0 | 4 0 0 | 6 0 0 


Solution found :')

2 9 3 | 8 4 7 | 5 6 1 
7 5 8 | 3 6 1 | 2 4 9 
6 1 4 | 2 5 9 | 7 8 3 
------+-------+------
9 7 5 | 1 2 4 | 8 3 6 
3 2 1 | 6 8 5 | 4 9 7 
4 8 6 | 9 7 3 | 1 2 5 
------+-------+------
8 4 9 | 7 1 6 | 3 5 2 
1 6 2 | 5 3 8 | 9 7 4 
5 3 7 | 4 9 2 | 6 1 8 

Algorithm: BFS
Total time: 138 ms
Total expanded states: 18627
